tests/Yaml/Form/FormYamlProcessorTest.php

```diff
@@ -2,22 +2,29 @@
 
 namespace Ssch\TYPO3Rector\Tests\Yaml\Form;
 
-use Ssch\TYPO3Rector\Tests\Application\ApplicationFileProcessor\AbstractApplicationFileProcessorTest;
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
 
-final class FormYamlProcessorTest extends AbstractApplicationFileProcessorTest
+final class FormYamlProcessorTest extends AbstractRectorTestCase
```

Comment:
Best pratice for testing: test case should inherit from `AbstractRectorTestCase` and test some change of specific file. This will make sure it's easy to add failing test case and fix it :+1:

Reply:
Did it for every Processors. This will break yours. Sorry.

Reply:
Great!

This PR is now just for probe purposes anyway :) most of work is the processors one.